src/ui/controls/RadioListControl.tsx

```diff
@@ -40,6 +40,15 @@ export class RadioListControl extends React.Component<IStringControlProps, void>
   }
 
   /** @override */
```

Comment:
Done.  Interesting... thanks for pointing that out. Updated. And I'll create separate PR to flush out the unnecessary overrides in other the classes.